frontend/src/components/App/Home/index.tsx

```diff
@@ -0,0 +1,405 @@
+import { Icon, InlineIcon } from '@iconify/react';
```

Comment:
This should be broken down into separate files for each component within the Home/ folder. For example a ContextMenu.tsx